bop_toolkit_lib/visualization.py

```diff
@@ -122,6 +134,13 @@ def vis_object_poses(
     :param vis_rgb_resolve_visib: Whether to resolve visibility of the objects
       (i.e. only the closest object is visualized at each pixel).
     """
+    # check if CameraModel is used and if hand_tracking_toolkit is available
+    if isinstance(K, CameraModel) and not htt_available:
```

Comment:
Does this test make sense here? CameraModel should not be defined here if htt_available is False.

Reply:
However late at Line 177 I'd write 

```
if htt_available and isinstance(K, CameraModel):
    ....
```